app/models/concerns/inventory_computer.rb

```diff
@@ -85,11 +93,13 @@ def initialize(model)
       end
 
       def compute
-        as_of(Time.now)
+        as_of(nil)
```

Comment:
Are we using this anywhere?

Reply:
Yes this is used to compute quantities _"as of"_ a certain time. I just modified it not to include a timestamp unless needed (default behaviour)